mlir/lib/Dialect/Bufferization/Transforms/BufferUtils.cpp

```diff
@@ -138,8 +140,8 @@ bufferization::getGlobalFor(arith::ConstantOp constantOp, uint64_t alignment,
       alignment > 0 ? IntegerAttr::get(globalBuilder.getI64Type(), alignment)
                     : IntegerAttr();
 
-  BufferizeTypeConverter typeConverter;
```

Comment:
Maybe I dont understand this part of bufferizer. Could you please add a comment why s/`typeConverter.convertType(type)`/getMemRefTypeWithStaticIdentityLayout(type)/' is correct.